diff --git a/src/quick/items/qquicktableview.cpp b/src/quick/items/qquicktableview.cpp
index 04714b2310..28301de3f3 100644
--- a/src/quick/items/qquicktableview.cpp
+++ b/src/quick/items/qquicktableview.cpp
@@ -7125,9 +7125,7 @@ void QQuickTableViewResizeHandler::updateState(QEventPoint &point)
 void QQuickTableViewResizeHandler::updateDrag(QPointerEvent *event, QEventPoint &point)
 {
     auto tableView = static_cast<QQuickTableView *>(parentItem()->parent());
-#if QT_CONFIG(cursor)
     auto tableViewPrivate = QQuickTableViewPrivate::get(tableView);
-#endif
 
     switch (m_state) {
     case Listening:
