diff --git a/src/qml/CMakeLists.txt b/src/qml/CMakeLists.txt
index 2763f2187..424c4ee3e 100644
--- a/src/qml/CMakeLists.txt
+++ b/src/qml/CMakeLists.txt
@@ -473,7 +473,8 @@ add_custom_command(
     DEPENDS
         "${qml_build_dir}/plugins.qmltypes"
     COMMAND
-        ${CMAKE_COMMAND} -E copy_if_different
+        #${CMAKE_COMMAND} -E copy_if_different
+        cp -f
         "${qml_build_dir}/plugins.qmltypes"
         "${qml_root_dir}/builtins.qmltypes"
     COMMENT "Copying builtins.qmltypes"
@@ -486,7 +487,8 @@ add_custom_command(
     DEPENDS
         "${qml_build_dir}/jsroot.qmltypes"
     COMMAND
-        ${CMAKE_COMMAND} -E copy_if_different
+        #${CMAKE_COMMAND} -E copy_if_different
+        cp -f
         "${qml_build_dir}/jsroot.qmltypes"
         "${qml_root_dir}/jsroot.qmltypes"
     COMMENT "Copying jsroot.qmltypes"
